contracts/core/Dispatcher.sol

```diff
@@ -724,12 +713,10 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, ReentrancyGuard, IDi
         bool feeEnabled,
         ChannelEnd calldata counterparty
     ) internal {
-        // Register port and channel mapping
-        // TODO: check duplicated channel registration?
-        // TODO: The call to `Channel` constructor MUST be move to `openIbcChannel` phase
-        //       Then `connectIbcChannel` phase can use the `version` as part of `require` condition.
+        // We don't need to check that a channel isn't already present between a portAddress and a chanenlId since
+        // polymer chain verification should prevent double registration of the same channel.
```

Comment:
maybe add a note about the `feeEnabled` case here?

Reply:
good one - added comments, and committed it as acknowledging (but not fixing) the lack of feeEnabled validation